### ISSUE TITLE
LetterAvatar: How can i use in <p:dataTable/>?

### ISSUE THREAD

**Issue by marcelocaser** [primefaces-extensions/primefaces-extensions.github.com#695]
Problem using the "pe:letterAvatar" component with the "p:dataTable". 

As reported in the post https://forum.primefaces.org/viewtopic.php?f=14&t=58781&p=175013#p175013

![image](https://user-images.githubusercontent.com/4399574/57135460-7e5bec00-6d77-11e9-8ac9-eb288058d189.png)

**Comment by melloware** [comment]
I have verified the issue with this test case: 
[pfe-la.zip](https://github.com/primefaces-extensions/primefaces-extensions.github.com/files/3141695/pfe-la.zip)

Run "mvn clean jetty:run-exploded" and navigate to http://localhost:8080/primefaces-test/test.xhtml to see the issue.

**Comment by melloware** [comment]
OK I see the problem, Just need to come up with a clever way to fix it.

For now if you need a quick fix just do this to your datatable:
```xml
<p:ajax event="page" oncomplete="LetterAvatar.transform();" />
```
You would have to do similar for sorting and filtering.    I will come up with a real fix though.

**Comment by marcelocaser** [comment]
Thanks for reply!

I tested the solution and it worked well! Thank you! I'm waiting for a more adequate solution!

**Comment by melloware** [comment]
Fixed with commit: https://github.com/primefaces-extensions/core/commit/b8b4ccc6aa0cc9fc83940a7b36ade16794189f62

I turned it into a more traditional PF widget and this will be released in 7.0.1
